Cython/Utility/ImportExport.c

```diff
@@ -334,12 +335,23 @@ static int ${import_star}(PyObject* m) {
     if (__Pyx_import_all_from(locals, m) < 0) goto bad;
     list = PyDict_Items(locals);        if (!list) goto bad;
 
+#if CYTHON_COMPILING_IN_LIMITED_API
+    for(i=0; i<PyList_Size(list); i++) {
+        name = PyTuple_Get_Item(PyList_Get_Item(list, i), 0); if (!name) goto bad;
+        item = PyTuple_Get_Item(PyList_Get_Item(list, i), 1); if (!item) goto bad;
+#else
     for(i=0; i<PyList_GET_SIZE(list); i++) {
         name = PyTuple_GET_ITEM(PyList_GET_ITEM(list, i), 0);
         item = PyTuple_GET_ITEM(PyList_GET_ITEM(list, i), 1);
+#endif
+
         utf8_name = PyUnicode_AsUTF8String(name);
         if (!utf8_name) goto bad;
+#if CYTHON_COMPILING_IN_LIMITED_API
+        s = PyBytes_AsString(utf8_name); if (!s) goto bad;
+#else
```

Comment:
Same question as above, should we define a new macro in ModuleSetupCode.c?

Reply:
That's in TypeConversion.c (in a section that's also unconditionally always included) as `__Pyx_PyBytes_AsString`